server/utils/presenterUtils.ts

```diff
@@ -316,4 +316,12 @@ export default class PresenterUtils {
       classes: 'govuk-tag--grey',
     }
   }
+
+  static formattedTime(time: ClockTime): string {
+    return `${time.twelveHourClockHour}:${time.minute.toString().padStart(2, '0')}${time.partOfDay}`
```

Comment:
A while back I started a refactoring branch with dates times etc when I was pairing with James C, but I don't think I'll end up finishing it - we started using the [`strftime` package](https://github.com/samsonjs/strftime) which you can pass a date string to, rather than having to convert this to a `clockTime` - might be a bit simpler?

Reply:
Yeah, perhaps that would be a good idea, but I'm not really keen on the idea of using this PR to integrate another package. I'll make a note of it, though.